src/flicking.js

```diff
@@ -282,24 +282,27 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		_setPadding: function(padding, build) {
 			var horizontal = this.options.horizontal;
 			var panel = this._conf.panel;
+			var paddingSum = padding[0] + padding[1];
+			var cssValue = {};
 
 			panel.size = this.$wrapper[
-				horizontal ? "outerWidth" : "height"
-			]() - (padding[0] + padding[1]);
+					horizontal ? "outerWidth" : "outerHeight"

```

Comment:
How about to change `outerXXX` to `innerXXX`?

Because content area does not include border.

![image](https://cloud.githubusercontent.com/assets/3255471/16765462/259f8d72-486f-11e6-97d5-f670e812e859.png)